TABSwiftLayoutTests/LayoutTests.swift

```diff
@@ -0,0 +1,270 @@
+//
+//  LayoutTests.swift
+//  TABSwiftLayoutTests
+//
+//  Created by Jonathan on 29/12/2017.
+//  Copyright © 2017 TheAppBusiness. All rights reserved.
+//
+
+import XCTest
+@testable import TABSwiftLayout
+
+class LayoutTests: XCTestCase {
+  
+  private let testPriority = LayoutPriority(rawValue: 751)
```

Comment:
is it sensible to always test with the same priority? if this testPriority was 1000, would it have still caught the bug you mentioned? might be worth having some tests that vary the priority, like you have for the other variables such as margins and alignments.

Reply:
There are a few that use `.required` but yep it would be better to have different values.

Reply:
Should I use random numbers for other constants in the tests? (eg EdgeMargins, offsets)